pkg/controllers/proxy/service_endpoints_sync.go

```diff
@@ -456,15 +463,13 @@ func (nsc *NetworkServicesController) cleanupStaleVIPs(activeServiceEndpointMap
 }
 
 func (nsc *NetworkServicesController) cleanupStaleIPVSConfig(activeServiceEndpointMap map[string][]string) error {
-
 	ipvsSvcs, err := nsc.ln.ipvsGetServices()
 	if err != nil {
 		return errors.New("Failed get list of IPVS services due to: " + err.Error())
 	}
 
 	// cleanup stale ipvs service and servers
 	glog.V(1).Info("Cleaning up if any, old ipvs service and servers which are no longer needed")
-	ipvsSvcs, err = nsc.ln.ipvsGetServices()
```

Comment:
This is already called a few lines above.

Reply:
Good catch